staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -38,10 +41,15 @@ type AuditLogOptions struct {
 	MaxSize    int
 
 	PolicyFile string
+
+	WebhookFile string
```

Comment:
I feel like this is going to have weird interactions with the feature flag and existing audit log flags. Seems like the behavior would have to be:

* No feature flag enabled -> don't need to set audit mode?
* Feature flag enabled -> need to set audit mode?